pkg/operator/ceph/cluster/cluster_external.go

```diff
@@ -118,6 +118,10 @@ func (c *ClusterController) configureExternalCephCluster(cluster *cluster) error
 	if err != nil {
 		return errors.Wrap(err, "failed to update csi cluster config")
 	}
+	err = csi.UpdateMonEndpointConfig(c.context.Clientset, c.namespacedName.Namespace, monEndpoints)
```

Comment:
For external clusters, if the mon endpoints change, the configmap is expected to be updated [here](https://github.com/rook/rook/blob/master/pkg/operator/ceph/cluster/mon/health.go#L809-L813) if the mons change. So I don't see why this is necessary here.